### ISSUE TITLE
Have you tested RCAN result on benchmark?

### ISSUE THREAD

**Issue by 1187697147** [xinntao/BasicSR#343]
Thanks for your amaing work.
I want to know if you have trained RCAN and the final result is close to original paper?
Wishsing for your reply!

**Comment by xinntao** [comment]
Now the RCAN is not completed. 

In my preliminary experiments, the training results are similar to the officially released repo.

![image](https://user-images.githubusercontent.com/17445847/101772760-c384fa00-3b26-11eb-9e29-87d40a60c6d3.png)

I may update it later.

**Comment by 1187697147** [comment]
To be honest, I personally think that many of the models in the last three years have not successfully reproduced the results in the paper, even if they provide the source code, which makes me feel very confused. If you can reproduce good models such as RCAN or SAN, this may be very valuable work in this field.
Thanks again for your reply!

**Comment by xinntao** [comment]
Thanks

You can also make contributions to BasicSR 😃

**Comment by zpkosmos** [comment]
@1187697147 I think there are some tricks in SR network training that have not been mentioned by the authors in their paper. For example, you could train your model first, and then load the model as the pre-trained model and train it once again. It can always improve the performance. This is like using the cosine learning rate.

**Comment by 1187697147** [comment]
@zpkosmos Okay, thanks for your reminding, I will try the tricks above.

**Comment by 1187697147** [comment]
@zpkosmos You mean finetune the model which is trained before?

**Comment by zpkosmos** [comment]
@1187697147 Yes, you can first train the model for 1000 epochs with a learning rate of 2e-4, and then train the model for another 1000 epochs with a learning rate of 1e-4. You can also change the validation set. Note that we are calculating the PSNR on the Y channel. Finally, 你可以跟我中文.

**Comment by 1187697147** [comment]
@zpkosmos ok, 谢谢
